sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -51,7 +53,11 @@ private[sql] object JDBCRelation extends Logging {
    * @param partitioning partition information to generate the where clause for each partition
```

Comment:
Add the other two `@param` for the new parameters?

Reply:
ok